### PR TITLE
Update ZilSwap

### PR DESCRIPTION
**Pull request by xyn1nja**

### CODE REVIEW

File: projects/zilswap/project.md [hunk 1]
```diff
@@ -1,10 +1,22 @@
...
 categories: exchange,nft,web3,dex,api,staking,defi,dapp,app,sdk,oracle,token,governance,dao,infra
-status: Live
+status: live
 twitter: https://twitter.com/ZilSwap
 website: https://zilswap.io/
-discord: https://discord.gg/SMzzQ2sUMN
-telegram: False
+discord: https://discord.gg/ZilSwap
 ---
+full_description:
```
Comment (AshIgnYeo) [inline projects/zilswap/project.md:12]: @xyn1nja over multiline values try wrapping it in quotes, This current file is breaking the structure.

**Review by AshIgnYeo — CHANGES_REQUESTED** [review]

File: projects/zilswap/project.md (hunk 1 above)
Comment (xyn1nja) [reply projects/zilswap/project.md]: updated, please review, thanks.

**Comment by Elliott-Green** [comment]
Need to test if this works on the site before merging.

**Comment by rrw-zilliqa** [comment]
@xyn1nja  - is this still relevant? please reopen if so (sorry - it's rather old ..)

**Pull request closed without merging**